src/shared/ui/paragraph/paragraph.module.scss

```diff
@@ -1,5 +1,6 @@
 .paragraph {
   margin: 0;
+  padding-top: 4px;
```

Comment:
This component is used in the whole project, this will change the markup of all components that use it. If you need to separate the text from the slider, use gap